Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -7044,6 +7044,13 @@ function revIDEStackFileVersion pStackFile
       end if
 end revIDEStackFileVersion
 
+function revIDEStackFileIsScriptOnly pStackFile
+   open file pStackFile for binary read
+   read from file pStackFile for 9
+   close file pStackFile
+   return it begins with format("%c%c%cscript",0xEF,0xBB,0xBF)
+end revIDEStackFileIsScriptOnly
```

Comment:
In the future we might support script-only stacks without BOMs, so I suggest:

1. Open the file for binary read
2. Read 4 bytes from the file; if they are "REVO", return false
3. Re-open the file for text read (don't specify encoding)
4. Read 1 line from the file; if the line doesn't match `^script ".*"` return false
5. Return true